src/mbgl/style/sources/geojson_source_impl.cpp

```diff
@@ -132,6 +138,7 @@ void GeoJSONSource::Impl::loadDescription(FileSource& fileSource) {
 
             loaded = true;
             observer->onSourceLoaded(base);
+            req.reset();
```

Comment:
This enables us to re-request the description when the previous request completed (main use case was: https://github.com/mapbox/mapbox-gl-native/pull/6751). 

Alternatively we could clear the request on [setURL](https://github.com/mapbox/mapbox-gl-native/blob/35e6e41873f64c33880140d7396bf9fe366e64c0/src/mbgl/style/sources/geojson_source_impl.cpp#L39).